extensions/shared/external-media/sources/request-external-access/index.js

```diff
@@ -0,0 +1,31 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+
+import PopupMonitor from './popup-monitor';
```

Comment:
Could we import from `@automattic/popup-monitor` instead?

Reply:
it looks like we'll import the whole `request-external-access`, so popup-monitor will be automatically pulled as a dependency